generators/python-v2/ast/src/TypeInstantiation.ts

```diff
@@ -86,6 +91,10 @@ export class TypeInstantiation extends AstNode {
         return new this({ type: "str", value });
     }
 
+    public static blockStr(value: string): TypeInstantiation {
+        return new this({ type: "block_str", value });
+    }
+
```

Comment:
Let's consolidate this functionality in the `str` helper above. Check out PHP's `multiline` option in [this method](https://github.com/fern-api/fern/blob/1908ee6716e2779e1af4a71865673397fb26f55b/generators/php/codegen/src/ast/Type.ts#L305), which prevents the caller from specifying `multiline` unless they actually care about it. That way there's a single `str` AST node, and it knows how to render itself based on the options provided.

I imagine we'd have something like:

```typescript
public static str(value: string, { block }: { block?: boolean } = {}): Type {
```